aws-push-notifications-pinpoint/src/main/java/com/amplifyframework/pushnotifications/pinpoint/AWSPinpointPushNotificationsActivity.kt

```diff
@@ -50,7 +49,8 @@ class AWSPinpointPushNotificationsActivity : Activity() {
         finish()
     }
 
-    private fun processIntent(action: Map<String, String>?): Intent? {
+    private fun getIntentAction(payload: NotificationPayload?): Intent? {
```

Comment:
I'd prefer that we keep NotificationPayload nonnull here. This would also make return type non-null

Reply:
problem is `getLaunchIntentForPackage` is nullable, so return type Intent is always going to be nullable.